test/general/__init__.py

```diff
@@ -19,3 +20,16 @@ def setup_solo_multiworld(world_type) -> MultiWorld:
     for step in gen_steps:
         call_all(multiworld, step)
     return multiworld
+
+
+def setup_solo_multiworld_without_calls(world_type: Type[World]) -> MultiWorld:
```

Comment:
could extend
`test/general/__init__.py`'s `def setup_solo_multiworld`
to
```python
def setup_solo_multiworld(world_type, steps=gen_steps) -> MultiWorld:
    ...
    for step in steps:
      ...
```

and pass an empty list instead of reimplementing the whole thing

Reply:
ah yeah good idea